specification/sql/resource-manager/readme.md

```diff
@@ -142,6 +142,7 @@ input-file:
   - Microsoft.Sql/preview/2021-05-01-preview/VirtualNetworkRules.json
   - Microsoft.Sql/preview/2021-05-01-preview/WorkloadClassifiers.json
   - Microsoft.Sql/preview/2021-05-01-preview/WorkloadGroups.json
+  - Microsoft.Sql/preview/2021-05-01-preview/ServerConnectionPolicies.json
```

Comment:
can we add it as alphabetical order? 

this poershell command will also generate the update list with alphabetical order.
> These can be regenerated by running the following PowerShell script from this readme file's folder: dir .\Microsoft.Sql\preview\2020-11-01-preview\ -File | Resolve-Path -Relative | % { " - $_".Replace("\", "/") }

Reply:
Yes will do.